src/Functions/FunctionJoinGet.h

```diff
@@ -77,13 +77,15 @@ class JoinGetOverloadResolver final : public IFunctionOverloadResolverImpl
     String getName() const override { return name; }
 
     FunctionBaseImplPtr build(const ColumnsWithTypeAndName & arguments, const DataTypePtr &) const override;
-    DataTypePtr getReturnType(const ColumnsWithTypeAndName & arguments) const override;
+    DataTypePtr getReturnType(const ColumnsWithTypeAndName &) const override { return {}; }
```

Comment:
Is it legal?

Reply:
`getReturnType` is supposed to be a private method for overload resolution. The return type can also be directly inferred in method `build` when Nullable is handled manually. I'll add some comment...